package.json

```diff
@@ -158,7 +159,15 @@
   "husky": {
     "hooks": {
       "pre-push": "npm run lint",
-      "pre-commit": "sh secrets-check.sh"
+      "pre-commit": "sh secrets-check.sh && lint-staged"
     }
+  },
+  "lint-staged": {
+    "**/*.(js|jsx)": [
+      "eslint --ext .js,.jsx --no-eslintrc -c .eslintrc.json --fix"
```

Comment:
Don't actually need to pass in .js or .jsx anymore, lint staged automatically does it. Same for the other globs